src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  books: booksReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger));
```

Comment:
- [ ] Nice use of redux logger here 🔥 keep rocking!!!.

Reply:
Thanks